installer/scripts/TelegrafTCPErrorTelemetry.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+countErr=$(grep -iF  "socket_writer" /var/opt/microsoft/docker-cimprov/log/telegraf.log | wc -l | tr -d '\n')
+echo "telegraf,AKS_RESOURCE_ID=${AKS_RESOURCE_ID} telegrafTCPWriteErrorCountTotal=${countErr}i"
```

Comment:
Don't we need AcsResourceName for aks-engine clusters?

Reply:
I am just adding this as a tag as i cannot emit a metric without a tag. this will be overridden.